CONTRIBUTING.md

```diff
@@ -222,20 +222,43 @@ options you would normally pass to `pytest`, for example:
 hatch run test:unit test/test_logging.py::TestSkipLoggingConfiguration::test_skip_logging_configuration
 ```
 
+### Run code quality checks locally
+
 We also use tools to ensure consistent code style, quality, and static type checking. The quality of your code will be
-tested by the CI, but once again, running the checks locally will speed up the review cycle. To check your code you
-can run:
+tested by the CI, but once again, running the checks locally will speed up the review cycle.
+
 
+To check your code type checking, run:
 ```sh
-hatch run test:lint
+hatch run test:type
 ```
 
-If the linters spot any error, you can fix it before checking in your code:
 
+To check your code format run:
 ```sh
-hatch run test:lint-fix
+hatch run format-check
 ```
 
+
+To format your code, you can run:
+```sh
+hatch run format
+````
+
+
+To check your code style according to linting rules run:
```

Comment:
@silvanocerza from what I understand we use both because `ruff` does not yet cover all the rules that `pylint` does, correct? maybe we can then define a "meta" command that runs both these commands?

Reply:
Correct. I wouldn't do it to be fair, the hope would be to remove `pylint` sooner than later with some faster linter like `pyright`.

Reply:
got it 👍🏽